clients/drcachesim/scheduler/scheduler.h

```diff
@@ -1485,6 +1504,21 @@ template <typename RecordType, typename ReaderType> class scheduler_tmpl_t {
     memtrace_stream_t *
     get_input_stream(output_ordinal_t output);
 
+    // Returns the possibly adjusted (for queued records) record ordinal for the current
```

Comment:
"possibly adjusted" is arguably an internal detail. It only means that the scheduler may have read ahead internally and if so would need to adjust the counts returned by the stream? Maybe it's not something the API user needs to be aware of.

Reply:
It's probably hard to see in the diff view, but these are actually not public, but protected: so they are more internal helpers.  Even so maybe your comment still applies as the stream class that calls these doesn't necessarily need to know what these are doing inside.  Removed.